js/main.js

```diff
@@ -866,6 +866,28 @@ function BlackboxLogViewer() {
         reader.readAsText(file);
     }
 
+    function exportCsv(file, options={}) {
+
+        function onSuccess(data) {
+            console.debug("CSV export finished in", (performance.now() - startTime) / 1000, "secs");
+            if (!data) {
+                console.debug("Empty data, nothing to save");
+                return;
+            }
+            let blob = new Blob([data], {type: 'text/csv'}),
+                e    = document.createEvent('MouseEvents'),
+                a    = document.createElement('a');
+            a.download = file || $(".log-filename").text() + ".csv";
+            a.href = window.URL.createObjectURL(blob);
+            //a.dataset.downloadurl =  ['text/csv', a.download, a.href].join(':');
```

Comment:
Please don't leave commented out code in, or at least add a comment stating why it was commented out, and when it should be re-enabled.